packages/rs-dpp/src/data_contract/data_contract.rs

```diff
@@ -130,17 +345,23 @@ impl DataContract {
             .into_btree_string_map()
             .map_err(ProtocolError::ValueError)?;
 
+        let id = data_contract_map
+            .remove_identifier(property_names::ID)
+            .map_err(ProtocolError::ValueError)?;
+
         let mutability = get_contract_configuration_properties(&data_contract_map)?;
         let definition_references = get_definitions(&data_contract_map)?;
         let document_types = get_document_types_from_contract(
+            id,
             &data_contract_map,
             &definition_references,
             mutability.documents_keep_history_contract_default,
             mutability.documents_mutable_contract_default,
         )?;
 
-        let protocol_version =
-            data_contract_map.remove_integer(property_names::PROTOCOL_VERSION)?;
+        let protocol_version = data_contract_map
+            .remove_optional_integer(property_names::PROTOCOL_VERSION)?
+            .unwrap_or(LATEST_VERSION);
```

Comment:
Don't we want whoever uses that method to pass the protocol version explicitly?

Reply:
It will be changed soon